kong/conf_loader/init.lua

```diff
@@ -626,17 +626,24 @@ end
 
 
 local function try_base64_decode(vals)
+  -- names that we should not attempt decoding
+  local decode_blacklist = {
+    system = "system"
```

Comment:
`system` was considered valid base64 for `+-z`. Not sure if there are other such words we should pay attention to.

Reply:
Resolved by moving the base64 decoding *after* the path & values checks.